msteams-platform/bots/how-to/conversations/workflow-bot-in-teams.md

```diff
@@ -179,11 +196,11 @@ You can create a new file `bot/src/cardActions/doSomethingActionHandler.js`:
     }
```

Comment:
Empty line after `module.exports = {`, can we remove it?

Reply:
Updated.